packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -329,6 +331,18 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			uploadTime: pending?.uploadTime,
 		});
 	}
+
+	public hasPendingStashedBlobs(): boolean {
+		if (
+			Array.from(this.pendingBlobs.values()).filter((e) => e.uploading === true).length > 0 &&
```

Comment:
I think this will block whenever there are blobs currently uploading, even if the stashed blobs have finished uploading. I wonder if it would make more sense to mark the individual stashed blob entries and only block if those haven't finished uploading.

Reply:
I considered that approach but I thought I could get rid of adding another property to the pending blobs by just adding the stashing property. I don't think I would get rid of the problems you mentioned here and your comment below so I'll try the individual marks approach